Gordon360/ApiControllers/HousingController.cs

```diff
@@ -355,7 +353,10 @@ public IHttpActionResult GetAllApartmentApplication()
                     return NotFound();
                 }
             }
-            return StatusCode(HttpStatusCode.Forbidden);
+            else
+            {
+                return StatusCode(HttpStatusCode.Forbidden);
```

Comment:
Rather than explicitly handlind authorization within the controller's route method, we prefer to encapsulate authorization within the StateYourBusiness Authroization filter. That way all the Authorization logic lives in one place and can be audited and updated consistently.

Reply:
Ah yes, we (referring to the ApartApp team) have discussed that and plan to do that I believe, my work on this branch so far has been solely focused on the logistics of interacting with the database, so I haven't yet taken the time to research any of the authorization stuff. This code there can be considered a placeholder and I will also add a card to our project board to pursue the authorization part soon, if there isn't a card for that already